packages/skeleton-svelte/src/lib/components/Listbox/ListboxItem.svelte

```diff
@@ -0,0 +1,92 @@
+<script lang="ts">
+	import { createId } from '$lib/internal/create-id.js';
+	import { getListboxContext } from './context.js';
+	import { getFirstElement, getLastElement, getNextElement, getPreviousElement } from '$lib/internal/elements.js';
+	import type { ListboxItemProps } from './types.js';
+
+	// Props
+	let {
+		value,
+		base = 'flex items-center gap-2 text-left',
+		padding = 'py-2 px-4',
+		radii = 'rounded',
+		selected = 'preset-filled',
+		classes,
+		leadBase,
+		leadClasses,
+		trailBase,
+		trailClasses,
+		children,
+		lead,
+		trail,
+		...attributes
+	}: ListboxItemProps = $props();
+
+	// Context
+	const context = getListboxContext();
+
+	// Local
+	const id = createId();
```

Comment:
I've followed up regarding the ID system in Discord. I think this might be a good point to talk through our approach to keyboard interaction for all components. Start implementing our shared conventions and paradigms around this, if not dedicated tooling (in-house, Zag, etc)

Reply:
Yeah, I really think Zag is really great, I definitely want to consider it.